### DISCUSSION TITLE
Show name of stashed applications or windows with the colors, so you can distinguish better

### DISCUSSION THREAD

**Discussion by warror988** [rxhanson/RectanglePro-Community#577] [Ideas]
Show name of stashed applications or windows with the colors, so you can distinguish better the stashed applications, actually is hard to distiguish if you have many stashed windows

**Comment by rxhanson** [comment]
Thanks for the feedback! This one is on the backlog along with app icons and app-specific colors in the color tabs, but I haven't made it there yet.

**Reply by warror988** [reply]
Please, work on it, the stash feature is great, but this need it urgently
